### PR TITLE
integrate vet confirmation and verification pingdom and statuspage

### PR DESCRIPTION
**Pull request by esubocz-va**
Related story: https://vajira.max.gov/browse/API-638
Integrate the Veteran Verification and Veteran Confirmation pingdom health checks with the status page.

### CODE REVIEW

File: pingdom-checks/veteran-verification.ping [hunk 1]
```diff
@@ -1,26 +1,35 @@
 #production
 pingdom save-check \
-  --template simple-resource \
+  --template https-public-200 \
```
Comment (bschofield-va) [inline pingdom-checks/veteran-verification.ping:3]: missing `resolution` template arg

Comment (bschofield-va) [reply pingdom-checks/veteran-verification.ping:3]: `-a resolution=1`

Comment (esubocz-va) [reply pingdom-checks/veteran-verification.ping:3]: 👍

Comment (esubocz-va) [reply pingdom-checks/veteran-verification.ping:3]: I fix.

**Comment by bschofield-va** [comment]
FYI @golavarr-va

**Pull request merged**